src/cascadia/PublicTerminalCore/HwndTerminal.cpp

```diff
@@ -186,10 +234,26 @@ void HwndTerminal::SendOutput(std::wstring_view data)
 
 HRESULT _stdcall CreateTerminal(HWND parentHwnd, _Out_ void** hwnd, _Out_ void** terminal)
 {
-    auto _terminal = std::make_unique<HwndTerminal>(parentHwnd);
+    auto _hostWindow = CreateWindowEx(
+        0,
+        L"static",

```

Comment:
I presume "static" is the class name... does it have to be this or could it be a more descriptive name to be seen under the spy tools?

Reply:
static is one of the predefined window classes. And as far as myself and the .NET experts can tell it has to be that.

Reply:
OK makes sense.